src/content/structured/components/text-input/code.mdx

```diff
@@ -283,6 +283,50 @@ export const snippetsValidation = [
   </div>
 </ComponentPreview>
 
+### With min and max
+
+In this example, the error message will change for inputted values below min and above max.
```

Comment:
In this example, an error message will display when the input is either below the `min` value or above the `max` value. This only effects numeric input fields.